certbot/compat/os.py

```diff
@@ -29,3 +29,15 @@
 
 # Clean all remaining importables that are not from the core os module.
 del ourselves, std_os, std_sys
+
+
+# The os.open function on Windows will have the same effect than a bare os.chown towards the given
+# mode, and will create a file with the same flaws that what have been described for os.chown.
+# So upon file creation, security.take_ownership will be called to ensure current user is the owner
```

Comment:
Does that mean, "after creating a file, you'll have to call take_ownership" or "hen you call the function to create a file, that function calls take_ownership"?

Reply:
Second option. I forgot that "will" can be seen as an intention in the future in english.

I make the comment more clear.